iocore/net/SSLNetVConnection.cc

```diff
@@ -973,8 +975,7 @@ SSLNetVConnection::sslStartHandShake(int event, int &err)
           return EVENT_DONE;
         } else {
           SSLConfig::scoped_config params;
```

Comment:
What is the point of `params`?

Reply:
Yes, no longer needed since sni_map_enable is removed.